web-common/src/features/models/workspace/ModelBody.svelte

```diff
@@ -53,6 +56,14 @@
   $: runtimeInstanceId = $runtime.instanceId;
   const updateModel = createRuntimeServicePutFileAndReconcile();
 
+  let limit = 150;
+
+  $: tableQuery = createQueryServiceTableRows(runtimeInstanceId, modelName, {
+    limit,
+  });
+
+  $: runtimeError = $tableQuery.error?.response.data;
```

Comment:
I believe the type for `response` is not present. There's already a TODO note in the file. For now we can just use `($tableQuery.error as any)?.response.data`

Reply:
Done